examples/simplejs/index.html

```diff
@@ -0,0 +1,58 @@
+<!doctype html>
+<html lang="en" data-framework="jquery">
+	<head>
+<style>
+
+</style> 	
+		<meta charset="utf-8">
+		<title>SimpleJS • TodoMVC</title>
+		<link rel="stylesheet" href="node_modules/todomvc-common/base.css">
+		<link rel="stylesheet" href="node_modules/todomvc-app-css/index.css">
+	</head>
+	<body>
+
+		<section id="todoapp">
+			<header id="header">
+				<h1>todos</h1>
+				<input id="new-todo" placeholder="What needs to be done?" autofocus>
+			</header>
+			<section id="main">
+				<input id="toggle-all" type="checkbox">
+				<label for="toggle-all">Mark all as complete</label>
+				<ul id="todo-list"></ul>
+			</section>
+			<footer id="footer"></footer>
+		</section>
+		<footer id="info">
+			<p>Double-click to edit a todo</p>
+			<p>Created by <a href="http://symbolclick.com">Yigal Irani</a></p>
+			<p>Part of <a href="http://todomvc.com">TodoMVC</a></p>
+		</footer>
+		<script id="todo-template" type="text/template">
+			<li #liClass data-id="#id" id="#id">
```

Comment:
Is `#liClass` a valid attribute? Is it backward compatible? Is it needed vs only standard attr names?